src/views/Home/LandingPage.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import "../../sass/landing.scss";
+import parisImage from "../../images/landing-destinations-paris.jpg";
+import tokyoImage from "../../images/landing-destinations-tokyo.jpg";
+import newYorkImage from "../../images/landing-destinations-newyork.jpg";
+import heroImage from "../../images/landing-hero-background.jpg";
+
+// Destination data array (AI Generated descriptions)
+const destinationsData = [
```

Comment:
This would be better extracted to another file within the same folder and imported